config/constants.go

```diff
@@ -6,6 +6,9 @@ const DefaultFormat = "png"
 // DefaultQuality is the default quality for processed images
 const DefaultQuality = 95
 
+// DefaultUserAgent is the default user-agent header to fetch images from URL with
+const DefaultUserAgent = ""
```

Comment:
we should add a default user agent with picfit and the current version

Reply:
@thoas hrm... referencing `application.Version` within `config` package results in circular imports. 

In order of (least to most) kludgery, but how do you feel about:
1. moving `application.Version` into `config.Version`.
2. passing `application.Version` through `config.Load(path string, version string)` and eventually into `config.DefaultConfig(version string)` (could store separately on the `Config{}` struct for future reference too)
3. hardcoding `application.Version` value in `config.DefaultUserAgent`

Not really happy with any of these, so if you have a better idea, suggest away.

Reply:
Hmm.. tough situation let me think about it :)

Reply:
Another solution is to move constants into sub-packages, but would want to modify for all packages with them for consistency.

e.g.
```go
import (
    applicationConstants "github.com/thoas/picsfit/application/constants"
    configConstants "github.com/thoas/picsfit/config/constants"
    engineConstants "github.com/thoas/picsfit/engine/constants"
    imageConstants "github.com/thoas/picsfit/image/constants"
)
```

Reply:
let's move ``application/constants.go`` to ``constants/constants.go`` at the root, it will solve the issue I think

Reply:
OK done